scripts/prepare_disk.sh

```diff
@@ -49,7 +49,7 @@ partition_resize()
 
         partition_end="$((start + size))"
         # sfdisk returns size in blocks, * (1024 / 512) converts to sectors
-        target_disk_end="$(($(sfdisk -s "${TARGET_DISK}" 2> /dev/null) * 2))"
+        target_disk_end="$(($(sfdisk --quiet -s "${TARGET_DISK}" 2> /dev/null) * 2))"
```

Comment:
Be consistent in using single letter options or full name options

Reply:
but not both options are always available!